src/elements/DipEdge.H

```diff
@@ -189,12 +298,25 @@ namespace impactx::elements
         amrex::ParticleReal m_psi; //! pole face angle in rad
         amrex::ParticleReal m_rc; //! bend radius in m
         amrex::ParticleReal m_g; //! gap parameter in m
+        amrex::ParticleReal m_R; //! scale length in m
+        amrex::ParticleReal m_K0; //! fringe field integral
+        amrex::ParticleReal m_K1; //! fringe field integral
         amrex::ParticleReal m_K2; //! fringe field integral
+        amrex::ParticleReal m_K3; //! fringe field integral
+        amrex::ParticleReal m_K4; //! fringe field integral
+        amrex::ParticleReal m_K5; //! fringe field integral
+        amrex::ParticleReal m_K6; //! fringe field integral
+        int m_model; //! model specification
+        Location m_flag; //! +1 for entry, or -1 for exit
```

Comment:
As a final update, I would like to make `m_model` and `m_flag` `AMREX_ENUM` as we do for `SpaceChargeAlgo`.

The reason is that I would like users to write in their input files / scripts a descriptive string or type instead of integers that one has to look up.

What would be good string values for each of those?
I assume:
- model: "linear", "order6" ?
- location: "entry", "exit"

Also would call `m_flag` simply `m_location`.

Reply:
I agree `m_location` works well, with `entry` and `exit` as string values.  For `m_model`, I would just use `linear` and `nonlinear`.  The question of "order" is subtle here, and unrelated to the values of K0-K6 above (which is where I assume the number 6 originated).